### ISSUE TITLE
Possible bug in using Batch normalisation

### ISSUE THREAD

**Issue by mrtpk** [Belval/CRNN#67]
- `training` parameter is not given while initialising `tf.layers.batch_normalization`.  The `training` parameter defaults to `False`. Hence while training the moving mean and moving variance won't be updated.

- Since this branch's tf api is <tf2, `tf.GraphKeys.UPDATE_OPS` and `train_op` have to be executed before training [[ref](https://www.tensorflow.org/api_docs/python/tf/compat/v1/layers/batch_normalization)]. I could not find these updations in the training code.

https://github.com/Belval/CRNN/blob/becd222c2495b109fb598cb4972e1b2cc8080aa6/CRNN/crnn.py#L163

**Comment by lulupanPPY** [comment]
boy，you saved this code
